octobot_trading/exchanges/config/ccxt_exchange_settings.py

```diff
@@ -0,0 +1,258 @@
+import octobot_trading.exchanges.parser as parser
+
+
+class CCXTExchangeTestStatus:
```

Comment:
Can you put it in a different file? We always try to separate classes in different modules

Reply:
done